### DISCUSSION TITLE
Ho to use correctly uvloop with Fast API and uvicorn?

### DISCUSSION THREAD

**Discussion by akonitx** [jgraph/drawio#3681] [Q&A] [answered]
I know when we install the`uvicorn[standard]` we use the`httptools` for parsing `HTTP` requests and `uvloop` for the asyncio event loop. But how to correctly check which event loop, as example the `uvloop` or the default `asyncio loop` we use in code?

And also i wanna know: should we declare:
```python
import asyncio
import uvloop
loop = uvloop.new_event_loop()
asyncio.set_event_loop(loop)
```
before instantiating a app from `FastAPI()` or after. Or moreover: we don't need to do this because `FastAPI()` uses the `uvloop` by default or not?

**Comment by davidjgraph** [comment, marked as answer]
huh?

**Reply by akonitx** [reply]
oh, sorry. Not i sends issue to drawio instead of FastAPI.
